sql-native/src/main/java/org/ballerinalang/sql/nativeimpl/ClientProcessor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.sql.nativeimpl;
+
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BObject;
+import io.ballerina.runtime.api.values.BString;
+import org.ballerinalang.sql.Constants;
+import org.ballerinalang.sql.datasource.SQLDatasource;
+
+import java.util.UUID;
+
+/**
+ * This class implements the native methods for the clients to be used.
+ *
+ * @since 1.2.0
+ */
+public class ClientProcessor {
+
+    private ClientProcessor() {
+    }
+
+    public Object createSqlClient(BObject client, BMap<BString, Object> sqlDatasourceParams,
```

Comment:
Don't we need to make them `static`?

Reply:
yes, we need. I will add. (made some changes later)